packages/network/src/api/index.ts

```diff
@@ -25,9 +26,27 @@ export class NetworkClient extends BaseApiClient {
     return process.env.DEFENDER_API_URL || 'https://defender-api.openzeppelin.com/';
   }
 
-  public async listSupportedNetworks(params?: ListNetworkRequestOptions): Promise<Network[]> {
+  public async listSupportedNetworks(
+    params: ListNetworkRequestOptions & { includeDefinition: true },
+  ): Promise<NetworkDefinition[]>;
+  public async listSupportedNetworks(
+    params?: ListNetworkRequestOptions & { includeDefinition?: false | undefined },
```

Comment:
Seems this would be equivalent:
```suggestion
    params?: ListNetworkRequestOptions & { includeDefinition?: false },
```